app/src/main/res/values/colors.xml

```diff
@@ -7,4 +7,5 @@
     <color name="minako">#58a6dc</color>
     <color name="vivid_rabbit">#7e6ca8</color>
     <color name="uo_color1">#FFA817</color>
+    <color name="transparent_black">#00000000</color>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tumugin/VirtualChemLight/blob/1971ec9a7dd14c688ccad41fe90811b42f4b00bb/app/src/main/res/values/colors.xml#L10"></span>The resource <code>R.color.transparent_black</code> appears to be unused</td>
    </tr>
  </tbody>
</table>